packages/logging/src/handlers/log.handler.ts

```diff
@@ -7,37 +7,86 @@ import {ServiceDefinitionTagEnum, tag, TracingContext, InternalContainerParamete
 import {LogHandlerInterface} from "../interfaces/log-handler.interface";
 import {Utils} from "../utils/utils";
 
+/**
+ * The LogHandler to use when we want to output some logs.
+ * This handler makes sure that only the right level of logs are outputted, stacks logs, and logs with different loggers.
+ * It is registered with the tag LogHandlerInterface so that it can be injected as a LogHandlerInterface to facilitate mocking.
+ */
 @tag("LogHandlerInterface")
 @injectable()
 export class LogHandler implements LogHandlerInterface {
 
+  /**
+   * @param loggers The loggers to use to output the logs. All services with the tag ServiceDefinitionTagEnum.Logger will be automatically injected here.
+   * @param logSeverityLevelConfiguration The severity from which to start logging the logs.
+   * @param activateDiagnostics Whether or not the outputted logs should contain the diagnostic part.
```

Comment:
We should say here that this is an intensive process and can dramatically reduce the performance of the code.